lib/suspenders/app_builder.rb

```diff
@@ -113,10 +113,6 @@ def set_up_hound
       copy_file "hound.yml", ".hound.yml"
     end
 
-    def configure_newrelic
```

Comment:
I'm not a big fan of having suspenders dependencies that you have to know to configure to use. 

If we think monitoring is required and we think skylight is the tool for that, then should we make it impossible to deploy without it? That was my thinking by suggesting the configuration file. While It's not *needed*, I think it is possible to use one and would let us use `ENV.fetch` so we get errors if we forget to set it up.